bigquery/google/cloud/bigquery/_http.py

```diff
@@ -38,4 +49,21 @@ class Connection(_http.JSONConnection):
     API_URL_TEMPLATE = "{api_base_url}/bigquery/{api_version}{path}"
     """A template for the URL of a particular API call."""
 
-    _EXTRA_HEADERS = {_http.CLIENT_INFO_HEADER: _CLIENT_INFO}
+    @property
+    def USER_AGENT(self):
+        return self._client_info.to_user_agent()
+
+    @USER_AGENT.setter
+    def USER_AGENT(self, value):
+        self._client_info.user_agent = value
```

Comment:
These aren't constants, so why uppercase?

Reply:
For backwards compatibility.

Reply:
There's a `USER_AGENT` on the superclass, which we want to override the behavior of.